yfinance/base.py

```diff
@@ -370,9 +370,12 @@ def cleanup(data):
         except Exception:
             pass
 
-        if not isinstance(data.get('summaryDetail'), dict):
-            # For some reason summaryDetail did not give any results. The price dict usually has most of the same info
-            self._info.update(data.get('price', {}))
+        try:
+            if not isinstance(data.get('summaryDetail'), dict):
+                # For some reason summaryDetail did not give any results. The price dict usually has most of the same info
+                self._info.update(data.get('price', {}))
+        except Exception:
+            pass
```

Comment:
To be more specific, could wrap only relevant part in a try/except block.
Plus, I feel like we should try to catch a specific exception, not all kind of exceptions.

```suggestion
        if not isinstance(data.get('summaryDetail'), dict):
                try:
                        # For some reason summaryDetail did not give any results. The price dict usually has most of the same info
                        self._info.update(data.get('price', {}))
                except Exception:
                        pass
```